ChatSX/screens/ChatRoom/index.tsx

```diff
@@ -1982,6 +1983,32 @@ const ChatRoom = ({navigation, route}: ChatRoom) => {
     }
   };
 
+  // this function helps to share chatroom url
+  const onShare = async () => {
+    try {
+      let shareChatroomRequest = {
+        chatroomId: chatroomID,
+        domain: 'https://dummyurl.com',
+      };
+      let URL = shareChatroomURL(shareChatroomRequest);
+
+      const result = await Share.share({
+        message: URL,
+      });
+      if (result.action === Share.sharedAction) {
+        if (result.activityType) {
+          // shared with activity type of result.activityType
```

Comment:
why there is no code?

Reply:
Removed this code, as these checks are used to update the status of share activity by showing toast/alert/console